src/app/components/EmailNotificationSettingsDialog/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useMemo } from 'react';
+import { Trans } from 'react-i18next';
+import { ErrorBadge } from 'app/components/Form/ErrorBadge';
+import { discordInvite } from 'utils/classifiers';
+import { translations } from 'locales/i18n';
+
+interface ErrorMessageProps {
+  emailIsValid?: boolean;
+  hasUnconfirmedEmail?: boolean;
+  authError?: boolean;
+}
+
+export const ErrorMessage: React.FC<ErrorMessageProps> = ({
+  emailIsValid,
+  hasUnconfirmedEmail,
+  authError,
+}) => {
+  const renderMessage = useMemo(
+    () =>
+      authError
+        ? translations.emailNotificationsDialog.authErrorMessage
+        : !emailIsValid
+        ? translations.emailNotificationsDialog.invalidEmailWarning
+        : hasUnconfirmedEmail
+        ? translations.emailNotificationsDialog.unconfirmedEmailWarning
+        : '',
+    [authError, emailIsValid, hasUnconfirmedEmail],
+  );
+
+  return (
+    <ErrorBadge
+      className="tw-mt-0"
+      content={
+        <Trans
```

Comment:
This is incorrect - you're passing a link to the used translation but none of them can contain a link.